pallet.el

```diff
@@ -237,6 +238,32 @@ use `pallet--package-archives-copy' if USE-COPY is true."
       (message "Pallet: unpacking %s" package-name)
       (pallet--unpack-one package-name))))
 
+;; Command to remove unrequested packages
+(defun pallet-auto-remove-packages ()
+  "Remove packages not explicitly required in Cask file.
+
+This is useful if you synchronize your Cask file across multiple
+machines and rely on pallet to cynchronize your installed
+packages. After deleting a package on one machine and syncing
+your Cask file, run this command on your other machines to
+automatically remove that package on them as well."
```

Comment:
Well, that's not what it currently does, although it could be modified to do so. Currently it only uninstalls packages. Maybe I should make a separate function that first ensures that all packages requested in the Cask file are installed, and then calls this function, and then this function would become an internal one?